File: ghul.swift
var a = 1000
var b = 7
while a > b {
	var c = a - b
	print("\(a) - \(b) = \(c)")
	a = c
  //В swift для windows почему-то не работает sleep, пришлось искать обходной путь)
	var d = 100000000
	while d != 0 {
		d -= 1
	}
}
print("I'm Ghul!")

File: ghul.go
package main

import (
	"fmt"
	"time"
)

func main() {
	var a = 1000
	var b = 7
	for a > b {
		var c = a - b
		fmt.Println(a, " - ", b, " = ", c)
		a = c
		time.Sleep(30000)
	}
	fmt.Printf("i'm Ghul!")
}
